plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -98,149 +136,142 @@ export default function DataTable<D extends object>({
     gotoPage,
     preGlobalFilteredRows,
     setGlobalFilter,
-    setPageSize,
-    updateStickyTableSize,
-    state: { pageIndex, pageSize, globalFilter, tableWidth, tableHeight },
+    setPageSize: setPageSize_,
+    wrapStickyTable,
+    state: { pageIndex, pageSize, globalFilter: filterValue, sticky = {} },
   } = useTable<D>(
     {
       columns,
       data,
       initialState,
-      getStickyTableSize: useCallback(() => {
-        if (wrapperRef.current && tableRef.current) {
-          const width = tableRef.current.clientWidth;
-          const height =
-            wrapperRef.current.clientHeight -
-            (globalControlRef.current?.clientHeight || 0) -
-            (paginationRef.current?.clientHeight || 0);
-          return { tableWidth: width, tableHeight: height };
-        }
-        return null;
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-      }, [initialHeight, initialWidth]),
+      getTableSize: defaultGetTableSize,
+      globalFilter: defaultGlobalFilter,
+      ...moreUseTableOptions,
```

Comment:
Both `globalFilter` and `getTableSize` are overridable via `moreUseTableOptions`, not in use in Superset `TableChart` though.